django_project/core/settings/prod.py

```diff
@@ -68,7 +69,9 @@
         # This will add a client unique id in messages
         'raven.contrib.django.raven_compat.middleware.'
         'SentryResponseErrorIdMiddleware',
-    ) + MIDDLEWARE_CLASSES
+    )
+    # MIDDLEWARE_CLASSES is not defined...
```

Comment:
MIDDLEWARE_CLASSES is from Django 1.x it was deprecated and changed into MIDDLEWARE settings in Django 2.x.
I removed it in some files, but if you happen to found any that is using `MIDDLEWARE_CLASSES` like this, please upgrade/change to `MIDDLEWARE`
reference: https://docs.djangoproject.com/en/2.1/ref/settings/#std:setting-MIDDLEWARE
It says, it is a list as opposed to a tuple here.